lib/erlex.ex

```diff
@@ -506,6 +506,40 @@ defmodule Erlex do
     "#{deatomize(type)}#{do_pretty_print(types)}"
   end
 
+  defp do_pretty_print({:when_names, when_names, {:list, :paren, items}}) do
+    items = Enum.map(items, &trim_when_names(do_pretty_print(&1), when_names))
+    Enum.join(items, ", ")
```

Comment:
This can be an `Enum.map_join/3` and save an iteration.

```elixir
Enum.map_join(items, ", ", &trim_when_names(do_pretty_print(&1), when_names))
```